Containerfile.test

```diff
@@ -0,0 +1,28 @@
+# Main Deno Container
+FROM quay.io/chapeaux/deno:1.25.0 as deno
```

Comment:
Can we bump this up to 1.26.1 please? I pushed out that version to quay tonight.

Reply:
resolved in 521927b